Formula/minimesos.rb

```diff
@@ -0,0 +1,14 @@
+class Minimesos < Formula
+  desc "Testing infrastructure for Mesos frameworks"
+  homepage "https://minimesos.org/"
+  url "https://raw.githubusercontent.com/ContainerSolutions/minimesos/0.10.2/bin/minimesos"
```

Comment:
Would be better to use `https://github.com/ContainerSolutions/minimesos/archive/0.10.2.tar.gz` and just `bin.install "bin/minimesos"` if needed.